### ISSUE TITLE
Add support for localization

### ISSUE THREAD

**Issue by DonJayamanne** [julia-vscode/julia-vscode#2041]
Do we want to add support for localization.
With Jupyter, Python and some other extensions we just rely on the community to add support for location of other languges and the like.

Basically we just create a file named package.nls.json and most of the strings are hardcoded in there instead of having the text hardcoded in package.json.
E.g. the text `"title": "Julia: Restart Language Server"` would be hardcoded in another file.
And if we have a file named package.nls.zh-cn.json then based on locale VSCode will get the strings from tehre and merge into package.json.

As for strings added into the code base, that required a little bit more work, but once its setup, users can contribute to the commands.

We find this useful in the other extensions, and 99% of them are maintained by the community.

**Comment by DonJayamanne** [comment]
Again, this is not a priority at all, just a suggestion.

**Comment by davidanthoff** [comment]
Yeah, that would be cool, but I think at the same time I would say we wait with doing anything for that until someone shows up who actually commits to translating things into a specific language _and_ is willing to maintain that long term :) If someone does want to do that, please speak up here, it would be amazing!
